packages/sanity/exports/cli.ts

```diff
@@ -2,4 +2,5 @@ export {createCliConfig, defineCliConfig, getCliClient} from '@sanity/cli'
 
 export type {CliClientOptions, CliConfig} from '@sanity/cli'
 
+export type {studioEnvVariablesOptions} from '../src/_internal/cli/server/getStudioEnvironmentVariables'
 export {getStudioEnvironmentVariables} from '../src/_internal/cli/server/getStudioEnvironmentVariables'
```

Comment:
Prefer a single re-export per file:
```suggestion
export {getStudioEnvironmentVariables, type studioEnvVariablesOptions} from '../src/_internal/cli/server/getStudioEnvironmentVariables'
```